src/material/schematics/ng-generate/m3-theme/index.spec.ts

```diff
@@ -9,153 +9,11 @@ import {pathToFileURL} from 'url';
 import {generateSCSSTheme} from './index';
 import {Schema} from './schema';
 
-// Hue maps created from https://m3.material.io/theme-builder#/custom (using
-// #984061 as source color). Not using predefined M3 palettes since some neutral
-// hues are slightly off from generated theme.
-const testM3ThemePalette = new Map([
```

Comment:
I had to move this palette into a separate function, because something in the test was mutating it causing random failures. Also has the nice side-effect of moving it to the bottom of the file.